sys-cluster/easybuild-easyblocks/easybuild-easyblocks-4.6.2.ebuild

```diff
@@ -0,0 +1,29 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+SUB_P=$(ver_cut 1)
+
+DESCRIPTION="Provides a collection of easyblocks for EasyBuild."
+HOMEPAGE="
+	https://easybuild.io/
+	https://github.com/easybuilders/easybuild-easyblocks
+"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=sys-cluster/easybuild-framework-${SUB_P}[${PYTHON_USEDEP}]
+"
+BDEPEND="${RDEPEND}"
+
+src_prepare() {
+	# already there from easybuild_framework
+	rm easybuild/__init__.py
```

Comment:
`|| die`

Though this looks a bit weird, if this is the case wouldn't there also be a file conflict if the package is installed with pip? Or does pip not care about that?

Reply:
Yes, pip does not care:
```
~/.local/lib/python3.9/site-packages $ ls
base  easyblocks  framework  __init__.py  main.py  __pycache__  toolchains  tools
```